tests/conftest.py

```diff
@@ -474,6 +474,7 @@ def kill(self):
             """Clean up all built VMs"""
             for vm in self.vms:
                 vm.kill()
+            shutil.rmtree(tmp_path)
```

Comment:
would it better to use self.tmp_path for readability?

Reply:
done